packages/claims/src/claimsIssuer/claimsIssuer.ts

```diff
@@ -63,7 +64,8 @@ export class ClaimsIssuer extends Claims implements IClaimsIssuer {
     const curve: sjcl.SjclEllipticalCurve = ecc.curves.k256;
     const g = curve.G;
     const claim: IPrivateClaim = this.jwt.decode(token) as IPrivateClaim;
-    if (!(await this.verifySignature(token, claim.iss as string))) {
+    const proofVerifier = new ProofVerifier(await this.document.read(claim.signer));
+    if (!(await proofVerifier.verifyAuthenticationProof(token))) {
```

Comment:
I think this should be `verifyAssertionProof` because I think a PrivateClaim is would usually be a credential of credential of some kind, not an authentication proof. If you look at the DIDDoc spec, issuing VCs is done by a key with an assertion relationship: https://www.w3.org/TR/did-core/#assertion
Did you have some other perspective here?
```suggestion
    if (!(await proofVerifier.verifyAssertionProof(token))) {
```